internal/providers/terraform/azure/testdata/lb_outbound_rule_test/lb_outbound_rule_test.tf

```diff
@@ -26,13 +26,11 @@ resource "azurerm_lb" "example" {
 }
 
 resource "azurerm_lb_backend_address_pool" "example" {
-  resource_group_name = azurerm_resource_group.example.name
   loadbalancer_id     = azurerm_lb.example.id
   name                = "be-%d"
 }
 
 resource "azurerm_lb_outbound_rule" "rules" {
-  resource_group_name     = azurerm_resource_group.example.name
```

Comment:
Deprecated and removed, added v2 tests. https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-upgrade-guide#resource-azurerm_lb_outbound_rule